src/app/ai-result/page.tsx

```
@@ -69,13 +69,29 @@ export default function AiResultPage() {
-  }, [roadmapId, router, setEpics, setTasks, setTechStacks, setAssignments]);
+  }, [roadmapId, setEpics, setTasks, setTechStacks, setAssignments]);
 
+  // roadmapId が空の場合、API から現チームの最新ロードマップを取得してリカバリ
   useEffect(() => {
-    if (!roadmapId) {
-      router.replace("/requirements/new");
-      return;
-    }
+    if (roadmapId) return;
+    let cancelled = false;
+    setIsRecovering(true);
```

ESLint

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/app/ai-result/page.tsx:78:5
  76 |     if (roadmapId) return;
  77 |     let cancelled = false;
> 78 |     setIsRecovering(true);
     |     ^^^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  79 |     getLatestRoadmapForCurrentTeam().then((id) => {
  80 |       if (cancelled) return;
  81 |       if (id) {

(react-hooks/set-state-in-effect)